### ISSUE TITLE
Missing directories 

### ISSUE THREAD

**Issue by mrkrause** [ModelDBRepository/241165#2]
The original models distributed through the Blue Brain Project include empty `python_recordings` and `hoc_recordings` directories for each cell. These are useful because they allow the included scripts (run.py, createsimulation.hoc) to work right out of the box. However, git does not include empty directories, so they've vanished from this repository. 

It might be helpful to re-create these folders with a placeholder file in them, so that everything is immediately usable.

**Comment by pramodk** [comment]
@mrkrause : I haven't run this model but of curiosity - instead of creating empty directories with empty .placeholder fine, couldn't those directories be created on the fly? (by modifying the script a bit)

**Comment by ramcdougal** [comment]
I second the making them on the fly suggestion, e.g. using `os.makedirs`

**Comment by mrkrause** [comment]
Sure, that would work too. I was just trying to match the structure of the original models. 

It's an easy change on the Python side; I'm not totally sure how to do so in createsimulation.hoc: do you just call out to system("mkdir hoc_recording"), which I guess happens to work on every platform?

**Comment by pramodk** [comment]
> I'm not totally sure how to do so in createsimulation.hoc: do you just call out to system("mkdir hoc_recording"), which I guess happens to work on every platform

yeah, [h.system](https://nrn.readthedocs.io/en/latest/python/programming/system.html?highlight=hoc%20os%20system#system) comes to my mind. Or, may be [nrnpython](https://nrn.readthedocs.io/en/latest/hoc/programming/python.html?highlight=nrnpython#nrnpython).
